target/linux/ath79/dts/qcn5502_netgear_ex7300v2.dts

```diff
@@ -0,0 +1,219 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca956x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "Netgear EX7300v2";
+	compatible = "netgear,ex7300v2", "qca,qcn5500", "qca,qca9560";
```

Comment:
You didn't seem to get my renaming comment. A space should be added before v2 for model, and a hyphen for compatible. This would also require changes for file names and device name etc.

Reply:
Done